src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return carCost * days;
+  } else if (days >= 3 && days < 7) {
+    return (carCost * days) - smallDiscount;
```

Comment:
Check point 3 in checklist

Reply:
I dont use here 'else' with return statement, but I need to use 'else if' with return because of task condition, please correct me if Im mistaken

Reply:
`else/if` looks really huge and decreases the readability of your code. We don't use `else` or `else/if` unless we really need to.

So you can do it this way:
```suggestion
  if (days < 3) {
    return carCost * days;
  }  
  
  if (days >= 3 && days < 7) {
    return (carCost * days) - smallDiscount;
```